core/src/interfaces.rs

```diff
@@ -153,6 +153,7 @@ impl ProofType {
                 return sgx_prover::SgxProver::run(input, output, config)
                     .await
                     .map_err(|e| e.into());
+
```

Comment:
new line added, but not consistent formatting with the other provers above so better remove again

Reply:
removed.